pavelib/utils/timer.py

```diff
@@ -63,8 +63,9 @@ def timed(wrapped, instance, args, kwargs):  # pylint: disable=unused-argument
             log_message.update(exception_info)
 
             try:
-                if not exists(dirname(log_path)):
-                    os.makedirs(dirname(log_path))
+                log_dir = dirname(log_path)
+                if log_dir and not exists(log_dir):
+                    os.makedirs(log_dir)
```

Comment:
I tried that out, and it didn't seem like that code is manipulating the path enough to make it really worth it.